ui/ui-components/pages/profile/[id]/edit.tsx

```diff
@@ -411,17 +415,19 @@ export default function Page(props) {
                   </Form.Control>
                 </Form.Group>
               </Col>
-              <Col md={3}>
-                <div style={{ paddingTop: 34 }} />
-                <LoadingButton
-                  variant="outline-primary"
-                  size="sm"
-                  type="submit"
-                  disabled={loading}
-                >
-                  Add
-                </LoadingButton>
-              </Col>
+              {process.env.GROUPAROO_RUN_MODE === "config" ? null : (
```

Comment:
I think this should be `process.env.GROUPAROO_UI_EDITION === "config"`

Reply:
(GROUPAROO_RUN_MODE) is on the server-side

Reply:
oops!  yep.